### ISSUE TITLE
Male or Female?

### ISSUE THREAD

**Issue by RhodesPeter** [a-la-node/hapi-project#14]
We need to create a user input field so we know weather to generate a male or female identity.

**Comment by emilyb7** [comment]
Agree! While I think that many men would like to assume a woman's identity in their new lives and vice-versa, I personally think that it would be difficult for me to fit in as Bradley Rogers when I move to Santa Monica.
